# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openoffice-org/OOo-use_bin_hsqldb.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Submitted By: DJ Lucas <dj AT linuxfromscratch DOT org>
Date: 2007-06-24
Initial Package Version: 2.2.1
Upstream Status: Not applicable
Origin: Self
Description: Uses the pre-built jar instead of recompiling hsqldb, which will
not compile with JDK-6.  Upstream hsqldb is targeting 1.9.0 which is currently
incompatible with 1.8.0.x.

--- OpenOffice-2.2.1-orig/hsqldb/makefile.mk	2006-12-01 17:33:12.000000000 +0000
+++ OpenOffice-2.2.1/hsqldb/makefile.mk	2007-06-22 23:29:30.000000000 +0000
@@ -67,13 +67,13 @@
 #                     src$/org$/hsqldb$/persist$/ScaledRAFileInJar.java \
 #                     src$/org$/hsqldb$/test$/TestCollation.java
 
-.IF "$(JAVACISGCJ)"=="yes"
-JAVA_HOME=
-.EXPORT : JAVA_HOME
-BUILD_ACTION=$(ANT) -Dbuild.label="build-$(RSCREVISION)" -Dbuild.compiler=gcj -f $(ANT_BUILDFILE) jar
-.ELSE
-BUILD_ACTION=$(ANT) -Dbuild.label="build-$(RSCREVISION)" -f $(ANT_BUILDFILE) jar
-.ENDIF
+#.IF "$(JAVACISGCJ)"=="yes"
+#JAVA_HOME=
+#.EXPORT : JAVA_HOME
+#BUILD_ACTION=$(ANT) -Dbuild.label="build-$(RSCREVISION)" -Dbuild.compiler=gcj -f $(ANT_BUILDFILE) jar
+#.ELSE
+#BUILD_ACTION=$(ANT) -Dbuild.label="build-$(RSCREVISION)" -f $(ANT_BUILDFILE) jar
+#.ENDIF
 
 .ENDIF # $(SOLAR_JAVA)!= ""
 
